tests/test_sklearn_histgbdt_converters.py

```diff
@@ -0,0 +1,85 @@
+"""
+Tests Sklearn HistGradientBoostingClassifier converters.
+"""
+import unittest
+import warnings
+
+import numpy as np
+import torch
+from sklearn.ensemble._hist_gradient_boosting.gradient_boosting import HistGradientBoostingClassifier
+
+import hummingbird.ml
+from tree_utils import gbdt_implementation_map
+
+
+class TestSklearnHistGradientBoostingClassifier(unittest.TestCase):
+    # Check tree implementation
+    def test_gbdt_implementation(self):
+        warnings.filterwarnings("ignore")
+        X = np.random.rand(10, 1)
+        X = np.array(X, dtype=np.float32)
+        # y = np.random.randint(2, size=10)
+

```

Comment:
nit: Did we miss the actual test here? Currently, it doesn't test any. Can we please add any assertions or remove this method?